### PR TITLE
Add new methods in Main.rb files (to Implement Game and Author classes)

### LINKED ISSUE #20: [2.5pt] Add new methods in Main.rb files (to Implement Game and Author classes)
- The following options should be available:
  - List of games
  - List all authors (e.g. 'Stephen King')
  - Add a game

### PR DESCRIPTION
**Pull request by medaminedev66**
Add the following option methods to the project:
- `list_games`
- `list_authors`
- `add _game`
- `enter_date` gets the year, month, and day and returns a date object.

close #20

### CODE REVIEW

File: add_options.rb [hunk 1]
```diff
@@ -41,4 +44,33 @@ def add_music_album
     print ENTR_MSG
     gets
   end
+
+  def add_game
+    puts 'Pleae enter the publish date of the game:'
```
Comment (rpire) [inline add_options.rb]: Please be careful not to print typos into the UI. ☝️

File: add_options.rb [hunk 2]
```diff
@@ -41,4 +44,33 @@ def add_music_album
...
+    publish_date = enter_date
+    puts 'Is it multi_player:'
+    multi_player = gets.chomp
+    puts 'The date of the last player:'
+    last_player_at = enter_date
+    @games << Game.new(publish_date, multi_player, last_player_at)
+    puts 'Game is added succussfully'
```
Comment (rpire) [inline add_options.rb]: The UI must show attention to detail. Please be careful not to print typos that the user can see when using the program. 🔎

File: add_options.rb [hunk 3]
```diff
@@ -41,4 +44,33 @@ def add_music_album
...
+    begin
+      date = Date.new(year, month, day)
+    rescue StandardError
+      puts "Be sure that you entered the correct date format\n
+      Please repeat again:"
+      enter_date
+    else
+      date
+    end
+  end
```
Comment (rpire) [inline add_options.rb:75]: When I tried this method, it made me enter the date twice. I entered the exact same date on both occasions to make sure I didn't make a mistake. Please, review this method to confirm if you are having the same issue.

**Review by rpire — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi, @medaminedev66 👋

You did a fantastic job implementing the `add_game` method. It's working as intended, and it's adding new games every time it's called. 🤓

However, some changes need to be implemented before merging into the `dev` branch. Please, take a look at the comments below for more details.

Cheers! 👍

**Review by aliabbani — APPROVED** [review]
Hi @medaminedev66 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥇 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: add_options.rb (hunk 3 above)
Comment (medaminedev66) [reply add_options.rb:75]: Thanks for reviewing my pull request. Please can you send to me a screenshot of the issue? which date have you used?

File: add_options.rb (hunk 1 above)
Comment (medaminedev66) [reply add_options.rb]: You are right. I haven't focused on the message. I just focused more on the performance of the method. I will work on it.

**Review by rpire — APPROVED** [review]
## Approved 🙌

You did a nice work implementing the required changes. Please proceed to merge these changes into the `dev` branch. 🚀

Cheers and happy coding! 🤓

**Pull request merged**